examples/Q-TCP/q_tcp.py

```diff
@@ -213,7 +216,7 @@ def qubit_send_w_retransmission(host, q_size, receiver_id, checksum_size_per_qub
             # encode logical qubit
             q.cnot(err_1)
 
-            dummy, ack_received = host.send_qubit(receiver_id, q, await_ack=True)
+            _, ack_received = host.send_qubit(receiver_id, q, await_ack=True)
             # messages = host.get_classical(receiver_id, wait=WAIT_TIME)
```

Comment:
Unused code can be removed. 
To me when I see commented out code, it's a sign that the code is not yet finalized. 
Do you need this? Also see lines 22-24